lib/aposStyles.js

```diff
@@ -0,0 +1,69 @@
+export function stylesElements(widget) {
+  return widget._options?.aposStylesElements || null;
+}
+
+export function stylesAttributes(widget, additionalAttrs = {}) {
+  // Separate class and style from other additional attributes
+  const {
+    class: additionalClasses,
+    style: additionalStyle,
+    ...otherAttrs
+  } = additionalAttrs;
+
+  if (additionalClasses) {
+    if (!Array.isArray(additionalClasses) && typeof additionalClasses !== 'string') {
+      throw new Error('class must be a string or an array of strings');
```

Comment:
Let's not throw here, this breaks the entire app. Do console warnings or something.

Reply:
I disagree, I think it _should_ break it, and Astro handles well errors, and show the dev what's wrong:

<img width="1476" height="1059" alt="image" src="https://github.com/user-attachments/assets/424b7941-8abf-4fcc-bcf8-7b95cdda227f" />